### ISSUE TITLE
✅ 🐞 Bug Report: New container gap controls not carrying over the values set in previous version after updating to v3.16 - [ED-12211]

### ISSUE THREAD

**Issue by jofbe** [elementor/elementor#23794]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Since updating Elementor and Elementor PRO to 3.16.x, the gap in my containers, where I set them to 0, are back to some default value. In order to fix it, I have to edit the original page with Elementor, set the column and row values linked, then it does work (0 gets applied).

### Steps to reproduce

Edit the gap value to be 0 on a page with Elementor 3.15.x, while the global value is not 0, then update, and check the gap. It should still say 0 in the editor, but the global value should be showing in the content. Now set the column and row values to be linked, and the issue is gone.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.48 (Debian) mod_fastcgi/mod_fastcgi-SNAP-0910052141 OpenSSL/1.1.1g
	MySQL version: MySQL Community Server (GPL) v5.7.35
	PHP Version: 8.0.8
	PHP Memory Limit: 256M
	PHP Max Input Vars: 500000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.3.1
	Site URL: https://mydomain
	Home URL: https://mydomain
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 1
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.8.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields Multilingual
		Version: 2.0.5
		Author: OnTheGoSystems

	Advanced Custom Fields PRO
		Version: 6.2.1.1
		Author: WP Engine

	Art kártyás fizetés
		Version: 1.0.0
		Author: Somebody

	Custom Post Type UI
		Version: 1.14.0
		Author: WebDevStudios

	Duplicator Pro
		Version: 4.5.12.1
		Author: Duplicator

	Elementor
		Version: 3.16.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.16.0
		Author: Elementor.com

	Redirection
		Version: 5.3.10
		Author: John Godley

	SVG Support
		Version: 2.5.5
		Author: Benbodhi

	WordPress Importer
		Version: 0.8.1
		Author: wordpressdotorg

	WPML Media
		Version: 2.7.3
		Author: OnTheGoSystems

	WPML Multilingual CMS
		Version: 4.6.5
		Author: OnTheGoSystems

	WPML String Translation
		Version: 3.2.7
		Author: OnTheGoSystems


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Global Style Guide: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default


== Log ==
	
Log: showing 20 of 282023-07-04 11:14:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.13.1',
  'to' => '3.14.1',
)]
2023-08-22 16:08:45 [info] elementor::elementor_updater Started 
2023-08-22 16:08:45 [info] Elementor/Upgrades - _on_each_version Start  
2023-08-22 16:08:45 [info] Elementor/Upgrades - _on_each_version Finished 
2023-08-22 16:08:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.3',
)]
2023-08-22 16:08:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.14.1',
  'to' => '3.15.3',
)]
2023-09-13 11:42:13 [info] elementor::elementor_updater Started 
2023-09-13 11:42:13 [info] Elementor/Upgrades - _on_each_version Start  
2023-09-13 11:42:13 [info] Elementor/Upgrades - _on_each_version Finished 
2023-09-13 11:42:13 [info] Elementor/Upgrades - _v_3_16_0_container_updates Start  
2023-09-13 11:42:13 [info] Elementor/Upgrades - _v_3_16_0_container_updates Finished 
2023-09-13 11:42:13 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Start  
2023-09-13 11:42:13 [info] Elementor/Upgrades - _v_3_17_0_site_settings_updates Finished 
2023-09-13 11:42:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.3',
  'to' => '3.16.2',
)]
2023-09-13 11:42:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.3',
  'to' => '3.16.2',
)]
2023-09-13 11:54:37 [info] elementor-pro::elementor_pro_updater Started 
2023-09-13 11:54:37 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-09-13 11:54:37 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-09-13 11:54:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.16.0',
)]
2023-09-13 11:54:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.14.1',
  'to' => '3.16.0',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by mocana** [comment]
I also had an issue where my prior :root css values for changing container padding went out the window with this update.

**Comment by incircolo** [comment]
Here the same. Please stop to break sites with updates. You may anticipate that someone might have non-standard configurations.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core v3.16.3! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
